webapp/src/js/components/panoptes/genome/tracks/GenotypesChannel.js

```diff
@@ -480,6 +484,41 @@ const GenotypesControls = React.createClass({
     layoutGaps: React.PropTypes.bool,
   },
 
+  // FIXME: Ideally wanted to use <QueryString> to convert Query to String
```

Comment:
You can use the logic from <QueryString>, but you are best off extracting that out to a module (not a component) that both this and <QueryString> use.